packages/docs/content/foundation/layout-grid/responsive-design.mdx

```diff
@@ -9,13 +9,13 @@ The design system's layout, utility, and typography classes are built with respo
 
 Use the breakpoint prefixes `sm`, `md`, `lg`, and `xl` to apply styles at different viewport sizes. For example: `.ds-u-lg-display--none` sets an element's `display` property to `none` when the viewport is `1024px` or wider.
 
-| Prefix | Breakpoint          | Sass variable       | Description                  |
+| Prefix | Breakpoint          | CSS variable       | Description                  |
 | ------ | ------------------- | ------------------- | ---------------------------- |
-|        | `min-width(0px)`    | `$media-width-xs`   | Viewports `0px` and wider    |
-| `sm`   | `min-width(544px)`  | `$media-width-sm`   | Viewports `544px` and wider  |
-| `md`   | `min-width(768px)`  | `$media-width-md`   | Viewports `768px` and wider  |
-| `lg`   | `min-width(1024px)` | `$media-width-lg`   | Viewports `1024px` and wider |
-| `xl`   | `min-width(1280px)` | `$media-width-xl`   | Viewports `1280px` and wider |
+|        | `min-width(0px)`    | `--media-width-xs`   | Viewports `0px` and wider    |
+| `sm`   | `min-width(544px)`  | `--media-width-sm`   | Viewports `544px` and wider  |
+| `md`   | `min-width(768px)`  | `--media-width-md`   | Viewports `768px` and wider  |
+| `lg`   | `min-width(1024px)` | `--media-width-lg`   | Viewports `1024px` and wider |
+| `xl`   | `min-width(1280px)` | `--media-width-xl`   | Viewports `1280px` and wider |
```

Comment:
These variables aren't usable yet as CSS custom props. Maybe we should leave this intact with some kind of disclaimer. They can be read, but we currently don't have a mechanism for sharing these values with teams making custom breakpoints without Sass.

Reply:
Same as above, these are available in the CSS as variables, but they don't do anything. However I could see them as being possibly useful as variables to reference in some cases?

Reply:
Maybe just a note that these can't be modified but can be utilized as references?

Reply:
Well they still are useful to teams as Sass variables for their custom breakpoints, so I'm saying maybe we shouldn't lose the documentation for the Sass variables just yet. Teams have custom breakpoints, and we don't currently have an alternative for them that is really thought out.

Reply:
Gotcha, updated this and the grid item to reflect Sass variables and added a note where it felt appropriate..